windows-release/uploadrelease.ps1

```diff
@@ -87,19 +88,28 @@ $dirs = gci "$build" -Directory
 if ($embed) {
     $dirs = ($dirs, (gi $embed)) | %{ $_ }
 }
+if ($sbom) {
+    $dirs = ($dirs, $sbom)
```

Comment:
```suggestion
    $dirs = ($dirs, $sbom) | %{ $_ }
```

Theoretically, the last bit shouldn't be needed (it serves to flatten the two arrays), but I'm sure I put it there on line 88/89 for a reason.